README.md

```diff
@@ -123,46 +123,46 @@ more documentation.
 
 ADS-B data from dump1090-docker can be
 [fed to ADS-B Exchange](https://www.adsbexchange.com/how-to-feed) with the help
-of [docker-adsbexchange](https://github.com/webmonkey/docker-adsbexchange).
+of [adsbexchange-docker](https://hub.docker.com/search?q=marcelstoer%2Fadsbexchange&type=image) images.
 
 * Using vanilla Docker
 
   1. Ensure dump1090 is running.
 
-  1. Fetch
-     [docker-adsbexchange](https://github.com/webmonkey/docker-adsbexchange)
-     and build Docker image.
+  1. Run [adsbexchange-docker-feed](https://github.com/marcelstoer/adsbexchange-docker).
 
      ```shell
-     git clone https://github.com/webmonkey/docker-adsbexchange.git
-     cd docker-adsbexchange
-     docker build -t webmonkey/adsbexchange:latest .
+     docker run --rm -d -e "INPUT=decoder:30005" —link dump1090:decoder --name adsbexchange-feed marcelstoer/adsbexchange-docker-feed:latest
      ```
+  1. Run [adsbexchange-docker-mlat](https://github.com/marcelstoer/adsbexchange-docker).
```

Comment:
I would say that this is optional. Some people might not want to feed their location with MLAT.